src/test/mod.rs

```diff
@@ -1,36 +1,34 @@
 pub mod jaegar;
```

Comment:
We have `"jaeger"` feature now:

```suggestion
#[cfg(feature = "jaeger")]
pub mod jaegar;
```

Reply:
Hm... but `src/test/jaeger.rs` is not actually connected to the jaegar tracer, so we probably shouldn't so this...

Reply:
I think is ok to leave jaeger as feature

Reply:
The problem with jaeger is that we don't actually support it in our `init_tracing_with_fallbacks`, so there is no way to init jaeger tracer using `telemetry-rust` API